test/unit/typescript/graph-tests.ts

```diff
@@ -39,4 +39,7 @@ async function myTest(client: Client): Promise<any> {
   let tokenString: string;
   tokenString = datastax.graph.t.id.toString();
   tokenString = datastax.graph.t.label.toString();
+  tokenString = datastax.graph.direction.in_.toString();
```

Comment:
What is the purpose of reassigning `tokenString` many times in a row without reading its value?

Reply:
TypeScript transpiles to js, so execution is done in javascript using the same runtime (Node.js).

We create integration tests in JavaScript, so runtime correctness is covered. For the TypeScript part, we have to make sure the API have the expected form. These type of tests are fairly common for non-native TypeScript projects (typescript definitions), see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0.1.450/types/cassandra-driver/cassandra-driver-tests.ts#L133-L137 for example